app/views/homepage/index.html.erb

```diff
@@ -1,3 +1,20 @@
+<%# 
+  The ga4_links_count variable is used to calculate the number of links within the homepage (ex. header and footer links) and this is passed to components rendered
+  on the homepage (e.g. services_and_information and government_activity) where it is used for analytics purposes.
+  
+  NB - Ministerial departments and other agencies "big number" links are hardcoded and therefore 2 is added to account for these links
+  (please see the _government_activity.html.erb template for further details)
+%>
+
+<% ga4_links_count = 
+  t("homepage.index.popular_links").length + 
+  t("homepage.categories").length + 
+  t("homepage.government_activity").length + 
+  t("homepage.index.more_links").length + 
+  t("homepage.index.promotion_slots").length +
+  2
```

Comment:
Oh there's the two! I see it now. Didn't see it because it was on a line by itself. Can I suggest a slight reformat?

```Ruby
<% ga4_links_count = 
  t("homepage.index.popular_links").length + 
  t("homepage.categories").length + 
  t("homepage.government_activity").length + 
  t("homepage.index.more_links").length + 
  t("homepage.index.promotion_slots").length + 2 # added to account for the two hardcoded big number links
%>
```

Reply:
Good shout, thanks! I'll make that change and squash the commits together